### ISSUE TITLE
CentOS Error installing Transformers

### ISSUE THREAD

**Issue by KimYar** [huggingface/transformers#7342]
## Environment info

     
- `transformers` version:
- Platform: CentOS
- Python version: 3.6.3
- PyTorch version (GPU?):1.6.0
- Tensorflow version (GPU?): tensorflow-gpu 2.3.0
- Using GPU in script?:
- Using distributed or parallel set-up in script?:
 
 @mfuntowicz 
@jplu 

## To reproduce

1. On a CentOS distribution try "pip install transformers" Python 3.6.3
pip install transformers

Looking in links: /cvmfs/soft.computecanada.ca/custom/python/wheelhouse/nix/avx2, /cvmfs/soft.computecanada.ca/custom/python/wheelhouse/nix/generic, /cvmfs/soft.computecanada.ca/custom/python/wheelhouse/generic
Ignoring pip: markers 'python_version < "3"' don't match your environment
Collecting transformers
  Using cached transformers-3.2.0-py3-none-any.whl (1.0 MB)
Collecting tokenizers==0.8.1.rc2
  Using cached tokenizers-0.8.1rc2.tar.gz (97 kB)
  Installing build dependencies ... done
  Getting requirements to build wheel ... done
    Preparing wheel metadata ... done
Processing /cvmfs/soft.computecanada.ca/custom/python/wheelhouse/generic/dataclasses-0.7-py3-none-any.whl
Processing /cvmfs/soft.computecanada.ca/custom/python/wheelhouse/generic/filelock-3.0.12-py3-none-any.whl
Requirement already satisfied: tqdm>=4.27 in /home/-/ENV/lib/python3.6/site-packages (from transformers) (4.49.0)
Processing /cvmfs/soft.computecanada.ca/custom/python/wheelhouse/generic/packaging-20.4-py2.py3-none-any.whl
Requirement already satisfied: numpy in /home/-/ENV/lib/python3.6/site-packages (from transformers) (1.19.1)
Processing /cvmfs/soft.computecanada.ca/custom/python/wheelhouse/generic/requests-2.24.0-py2.py3-none-any.whl
Processing /cvmfs/soft.computecanada.ca/custom/python/wheelhouse/generic/sacremoses-0.0.43-py3-none-any.whl
Processing /cvmfs/soft.computecanada.ca/custom/python/wheelhouse/nix/generic/sentencepiece-0.1.90-cp36-cp36m-linux_x86_64.whl
Requirement already satisfied: regex!=2019.12.17 in /home/-/ENV/lib/python3.6/site-packages (from transformers) (2019.11.1)
Requirement already satisfied: six in /home/-/ENV/lib/python3.6/site-packages (from packaging->transformers) (1.15.0)
Requirement already satisfied: pyparsing>=2.0.2 in /home/-/ENV/lib/python3.6/site-packages (from packaging->transformers) (2.4.7)
Processing /cvmfs/soft.computecanada.ca/custom/python/wheelhouse/generic/certifi-2020.6.20-py2.py3-none-any.whl
Processing /cvmfs/soft.computecanada.ca/custom/python/wheelhouse/generic/chardet-3.0.4-py2.py3-none-any.whl
Processing /cvmfs/soft.computecanada.ca/custom/python/wheelhouse/generic/idna-2.10-py2.py3-none-any.whl
Processing /cvmfs/soft.computecanada.ca/custom/python/wheelhouse/generic/urllib3-1.25.10-py2.py3-none-any.whl
Requirement already satisfied: joblib in /home/-/ENV/lib/python3.6/site-packages (from sacremoses->transformers) (0.16.0)
Requirement already satisfied: click in /home/-/ENV/lib/python3.6/site-packages (from sacremoses->transformers) (7.1.2)
Building wheels for collected packages: tokenizers
  Building wheel for tokenizers (PEP 517) ... error
  ERROR: Command errored out with exit status 1:
   command: /home/--/ENV/bin/python /home/--/ENV/lib/python3.6/site-packages/pip/_vendor/pep517/_in_process.py build_wheel /tmp/tmp_ut164h5
       cwd: /tmp/pip-install-7krg2wb2/tokenizers
  Complete output (38 lines):
  running bdist_wheel
  running build
  running build_py
  creating build
  creating build/lib
  creating build/lib/tokenizers
  copying tokenizers/__init__.py -> build/lib/tokenizers
  creating build/lib/tokenizers/models
  copying tokenizers/models/__init__.py -> build/lib/tokenizers/models
  creating build/lib/tokenizers/decoders
  copying tokenizers/decoders/__init__.py -> build/lib/tokenizers/decoders
  creating build/lib/tokenizers/normalizers
  copying tokenizers/normalizers/__init__.py -> build/lib/tokenizers/normalizers
  creating build/lib/tokenizers/pre_tokenizers
  copying tokenizers/pre_tokenizers/__init__.py -> build/lib/tokenizers/pre_tokenizers
  creating build/lib/tokenizers/processors
  copying tokenizers/processors/__init__.py -> build/lib/tokenizers/processors
  creating build/lib/tokenizers/trainers
  copying tokenizers/trainers/__init__.py -> build/lib/tokenizers/trainers
  creating build/lib/tokenizers/implementations
  copying tokenizers/implementations/__init__.py -> build/lib/tokenizers/implementations
  copying tokenizers/implementations/base_tokenizer.py -> build/lib/tokenizers/implementations
  copying tokenizers/implementations/bert_wordpiece.py -> build/lib/tokenizers/implementations
  copying tokenizers/implementations/byte_level_bpe.py -> build/lib/tokenizers/implementations
  copying tokenizers/implementations/char_level_bpe.py -> build/lib/tokenizers/implementations
  copying tokenizers/implementations/sentencepiece_bpe.py -> build/lib/tokenizers/implementations
  copying tokenizers/__init__.pyi -> build/lib/tokenizers
  copying tokenizers/models/__init__.pyi -> build/lib/tokenizers/models
  copying tokenizers/decoders/__init__.pyi -> build/lib/tokenizers/decoders
  copying tokenizers/normalizers/__init__.pyi -> build/lib/tokenizers/normalizers
  copying tokenizers/pre_tokenizers/__init__.pyi -> build/lib/tokenizers/pre_tokenizers
  copying tokenizers/processors/__init__.pyi -> build/lib/tokenizers/processors
  copying tokenizers/trainers/__init__.pyi -> build/lib/tokenizers/trainers
  running build_ext
  running build_rust
  /tmp/pip-build-env-1p_8fw9e/overlay/lib/python3.6/site-packages/setuptools/dist.py:452: UserWarning: Normalizing '0.8.1.rc2' to '0.8.1rc2'
    warnings.warn(tmpl.format(**locals()))
  error: Can not find Rust compiler
  ----------------------------------------
  ERROR: Failed building wheel for tokenizers
Failed to build tokenizers
ERROR: Could not build wheels for tokenizers which use PEP 517 and cannot be installed directly

**Comment by bugface** [comment]
The error message indicated that you need to first install Rust compiler (https://www.rust-lang.org/tools/install).
